tests/unit/components/results.js

```diff
@@ -0,0 +1,29 @@
+var sd = require('skin-deep'),
+    expect = require('chai').expect,
+    Results = require('../../../ui/components/results.jsx'),
+    fixture = require('../../fixtures/components/data.js');
+
+describe('Results', function() {
+  var tree;
+
+  beforeEach(function() {
+    tree = sd.shallowRender(<Results data={fixture}/>);
+  });
+
+  it('should not render the root suite', function() {
+    var suitesArray = tree.findNode('.results').props.children;
+
+    expect(suitesArray[0]).to.be.equal.null;
```

Comment:
Ehm... if you want to assert that something is null you can either do `to.be.null` or `to.equal(null)`.

Also, quite an interesting check to see if something is not rendered. I'm guessing you're returning `null` from the `render` method if the suite is the root one.

You could also filter the suites first and remove the root one and then render them. That way you can check that either the length of `.results`' children is shorter by one, or that the root suite is not included in the list of the `suite` prop sent to the children.

```js
expect(tree.findNode('.results').props.children).to.have.length(fixture.length - 1);
// or
expect(_.map(tree.findNode('.results').props.children, 'props.suite')).to.not.include(fixture[0]);
```

I prefer the first type of check combined with the test below (should render the suites array) that includes the check that the root suite is not rendered.